helpers.js

```diff
@@ -0,0 +1,55 @@
+function getStoredCredentials(keys) {
+    console.log(`Getting values for keys: ${JSON.stringify(keys)}`);
+
+    const values = [];
+    for (const key in keys){
+        values.push(new Promise((res) => {
```

Comment:
Why did you push the value and new promise to the value?

Reply:
that's a good question